src/main.rs

```diff
@@ -299,6 +299,7 @@ fn run() -> Result<()> {
     runner.execute(Step::Vim, "voom", || vim::run_voom(&base_dirs, run_type))?;
     runner.execute(Step::Node, "npm", || node::run_npm_upgrade(&ctx))?;
     runner.execute(Step::Node, "yarn", || node::yarn_global_update(run_type))?;
+    runner.execute(Step::Node, "pnpm", || node::pnpm_global_update(run_type))?;
```

Comment:
Does pnpm act on the same stuff as node and yarn (i.e. ./.node-modules and global modules)? In that case you'd probably want to run pnpm first, lest npm does all the work but slower.

Reply:
Hey @MCOfficer, yes `pnpm` is another javascript package manager like `npm` and `yarn`. However it doesn't make much sense to run them in a certain order, since they're separate package managers. `pnpm` and `npm` global updates are not linked and independent from one another.